### ISSUE TITLE
jsdoc-format rule fails if file starts with JsDoc comment.

### ISSUE THREAD

**Issue by Karankang007** [palantir/tslint#2505]
### Bug Report

- __TSLint version__: 5.0.0
- __TypeScript version__: 2.2.2
- __Running TSLint via__:  CLI 

#### TypeScript code being linted

```ts
/**
 * Singular representation of date time input field values
 * @param {Date} dateValue Date value
 * @param {Date} timeValue Time value
 */
export class DateTimeResult {
    constructor(private dateValue: Date, private timeValue: Date) {
    }
    ....
}
```

with `tslint.json` configuration:

```json
tslint:recommended
```

#### Actual behavior
ERROR: src/dateRangePicker/dateTimeResult.ts[1, 6]: asterisks in jsdoc must be aligned
ERROR: src/dateRangePicker/dateTimeResult.ts[2, 59]: asterisks in jsdoc must be aligned
ERROR: src/dateRangePicker/dateTimeResult.ts[3, 37]: asterisks in jsdoc must be aligned
ERROR: src/dateRangePicker/dateTimeResult.ts[4, 36]: asterisks in jsdoc must be aligned

#### Expected behavior
No JsDoc error

#### Workaround
Add a blank line at the top of the file.

**Comment by ajafff** [comment]
@Karankang007 I guess your file has a BOM? That would explain why the rule sees the asterisks in different columns

**Comment by Karankang007** [comment]
@ajafff  Yes, I am using VS 2015, which saves this with BOM by default. Either way, i think TSLint should ignore BOM characters when checking any rules.
